docs/source/usage.rst

```diff
@@ -2,11 +2,60 @@ Usage
 =====
 
 Running IAI MovieBot
-----------------------
+--------------------
+
+
+* A YAML configuration file is necessary to start the MovieBot, see example `config/moviebot_config.yaml`. 
+* Execute the command below to run the MovieBot.
 
 .. code-block:: shell
 
    $ python -m moviebot.run -c <path_to_config.yaml>
        
 
-Note: To create Slot-Values, execute the code once by setting `BOT: False` in the configuration file.
\ No newline at end of file
+Note: To create Slot-Values, execute the code once by setting `BOT: False` in the configuration file.
+
+Quickstart IAI MovieBot in the console
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To run MovieBot with default configuration execute the command below.
+
+.. code-block:: shell
+
+   $ python -m moviebot.run -c config/moviebot_config.yaml
```

Comment:
Nit: `-c config/moviebot_config.yaml` is not needed to load the default config.

Reply:
Done.